src/game_screen.cpp

```diff
@@ -49,23 +49,22 @@ void Game_Screen::CreatePicturesFromSave() {
 	}
 }
 
-void Game_Screen::Reset(bool is_load_savegame) {
+void Game_Screen::Reset() {
 	if (Main_Data::game_data.pictures.size() < pictures.size()) {
 		pictures.resize(Main_Data::game_data.pictures.size());
 	}
+
 	for (auto& p : pictures) {
 		if (p) {
 			p->Erase(false);
 		}
 	}
 
-	if (!is_load_savegame) {
-		data.flash_red = 0;
-		data.flash_green = 0;
-		data.flash_blue = 0;
-		data.flash_time_left = 0;
-		data.flash_current_level = 0;
-	}
+	data.flash_red = 0;
+	data.flash_green = 0;
+	data.flash_blue = 0;
+	data.flash_time_left = 0;
+	data.flash_current_level = 0;
 	flash_sat = 0;
 	flash_period = 0;
 
```

Comment:
Below this line (you can't see it unless you expand), we reset the tint if its negative.

Is it actually the case a negative value ever gets written to the tint chunks? If so it would be better to do this in fixup in liblcf.